packages/docs/content/blog/release-8.0.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Release 8.0
+date: 2023-09-18
+intro: Normalizing CSS, standardizing spacing & colors, a new Dropdown, and more
+---
+
+import hcHeaderImg from '../../src/images/hc-header-banner.png';
+
+We know that Open Enrollment is just around the corner for both Medicare.gov and HealthCare.gov teams, so this update is intentionally light. We don’t expect most teams to have to change anything in this major release. We did mark a few breaking changes because they are either “breaking” in principal or have the potential to cause unexpected changes. We lumped those changes into a major release to make sure teams are paying attention to the upgrade. We hope this release gives teams more flexibility and at the same time helps them create more consistent experiences as we head into fall.
+
+## What's new in 8.0
+
+### The HealthCare.gov header gets a USA banner
+
+<div className="ds-u-measure--wide ds-u-text-align--center ds-u-margin-top--3">
+  <img src={hcHeaderImg} width={685} className="ds-u-display--inline-block" alt="HealthCare.gov header with the USA banner above it" />
+</div>
+
+To make HealthCare.gov compliant with government standards and increase trust with consumers, its shared header component will now have the [USA Banner](/components/usa-banner/) built into it. To help coordinate a synchronized release of this banner on all applications at once, we’ve included a prop that teams can use to show or hide it.
+
+### The Medicare.gov footer says goodbye to its glossary link
+
+Medicare.gov is discontinuing their glossary page, so we’re removing the link from the footer. Teams that implement their own footer are responsible for removing it in their own applications.
```

Comment:
suggesting _its_ instead of _their_

Reply:
What would change to its?

Reply:
Medicare.gov is discontinuing its glossary page, so we’re removing the link from the footer. Teams that implement their own footer are responsible for removing it in their own applications.

Reply:
Oh, yeah, that makes sense